packages/cli/src/commands/serveBothHandler.js

```diff
@@ -0,0 +1,194 @@
+import path from 'path'
+
+import chalk from 'chalk'
+import execa from 'execa'
+
+import {
+  coerceRootPath,
+  createFastifyInstance,
+  redwoodFastifyAPI,
+  redwoodFastifyWeb,
+} from '@redwoodjs/fastify'
+import { getConfig, getPaths } from '@redwoodjs/project-config'
+
+export const bothExperimentalServerFileHandler = async () => {
+  logExperimentalHeader()
+
+  if (getConfig().experimental?.rsc?.enabled) {
+    logSkippingFastifyWebServer()
+
+    await execa(
+      'node',
+      [
+        '--conditions react-server',
+        './node_modules/@redwoodjs/vite/dist/runRscFeServer.js',
+      ],
+      {
+        cwd: getPaths().base,
+        stdio: 'inherit',
+        shell: true,
+      }
+    )
+    await execa(
+      'yarn',
+      ['node', path.join('dist', 'server.js'), '--enable-web'],
```

Comment:
```suggestion
      ['node', path.join('dist', 'server.js')],
```

Do we want to pass this `--enable-web` flag to the api side for the case of both and with RSC or SSR enabled? Passing this flag will make the api server use the web port and register the web plugin - so serve the static files and setup a proxy.